packages/components/image/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@contentful/f36-image",
+  "version": "4.0.0",
```

Comment:
Should be tagged as alpha?
You can use the script to make it easier, the script also adds it to the exclude file in changeset to avoid publishing it by mistake